### ISSUE TITLE
Proline support

### ISSUE THREAD

**Issue by mlocardpaulet** [Proteobench/ProteoBench#354]
We decided to only support outputs from Proline Studio. To do so, I need to:
- [ ] remove the Proline Web points
- [ ] find a way to identify what columns correspond to what sample. Column change depending of Proline parameters.
- [ ] document how to get parsable headers
- [ ] make a template for parameters output so that there is no personal information in the output, and all the information that we want to parse and save

**Comment by mlocardpaulet** [comment]
@RobbinBouwmeester and @julianu I need your opinion: 
There is now a parameter output template (.json) that the developers made. We can provide it so that users use it with ProlineStudio to export the parameters that we need (and not any personal information such as email addresses etc...). The plan is to provide a link to the file in the documentation so that people can download the .json. 
Now my question: where do I put the file? Shall I send it to @julianu to put it on the server? Any other idea?

**Comment by mlocardpaulet** [comment]
I guess that the inconvenient is that then you are the only one to have access? More work for you if the ProlineStudio developers request to modify the file etc.. 
Alternatively I guess that we could keep it somewhere in the github repo, but I am not really aware of pros/cons.
I'll do what you say :)

**Comment by julianu** [comment]
Yes, that would be the problem and the alternative. Actually, putting it into the repo (maybe even as another branch?) would also be a good way to keep it independent from updates. And the whole team has access.
Sounds better :)

**Comment by RobbinBouwmeester** [comment]
AFAIK this is fixed now
